inttest/ap-ha3x3/ha3x3_test.go

```diff
@@ -117,26 +119,17 @@ spec:
   timestamp: now
   commands:
     - k0supdate:
-        version: v0.0.0
-        forceupdate: true
+        version: ` + s.K0sUpdateVersion + `
         platforms:
           linux-amd64:
-            url: http://localhost/dist/k0s
+            url: http://localhost/dist/k0s-new
```

Comment:
There should probably be an assert in this test comparing the actual version of the new binary to the old version.

Reply:
Done